x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Response } from 'node-fetch';
+import { URL } from 'url';
+import {
+  AssetParts,
+  AssetsGroupedByServiceByType,
+  CategoryId,
+  CategorySummaryList,
+  KibanaAssetType,
+  RegistryPackage,
+  RegistrySearchResults,
+} from '../../../../common/types';
+import { epmConfigStore } from '../config';
```

Comment:
I believe the new pattern is to replace
```suggestion
import { configService } from '../../services';
```